src/compiler/utilities.ts

```diff
@@ -1281,402 +1281,400 @@ export function getInternalEmitFlags(node: Node): InternalEmitFlags {
-            es2020: emptyArray,
-        })),
-        RelativeTimeFormat: new Map(Object.entries({
-            es2020: [
-                "format",
-                "formatToParts",
-                "resolvedOptions",
-            ],
-        })),
-        Int8Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Uint8Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Uint8ClampedArray: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Int16Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Uint16Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Int32Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Uint32Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Float32Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Float64Array: new Map(Object.entries({
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        BigInt64Array: new Map(Object.entries({
-            es2020: emptyArray,
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        BigUint64Array: new Map(Object.entries({
-            es2020: emptyArray,
-            es2022: [
-                "at",
-            ],
-            es2023: [
-                "findLastIndex",
-                "findLast",
-            ],
-        })),
-        Error: new Map(Object.entries({
-            es2022: [
-                "cause",
-            ],
-        })),
-    }));
-}
+export const AllScriptTargetFeatures = new Map(Object.entries({

```

Comment:
There is a risk that some code mutates the values in this Map object, but its typed as ReadOnlyMap. I also tested that Object.freeze unfortunately does not guard Maps